docs/graphql/core/schema/custom-functions.rst

```diff
@@ -67,8 +67,11 @@ Example: Text-search functions
 
 Let's take a look at an example where the ``SETOF`` table is already part of the existing schema.
 
-In our article/author schema, let's say we've created and tracked a custom function, ``search_articles``,
-with the following definition:
+Let's say we have the following schema:
+
+author (id uuid, name text, city text, email text, phone integer, address text)
```

Comment:
1. Make this an SQL code-block.
2. The table that is actually used in the function later is `article` with fields `id`, `title` and `content`. `author` isn't used.